Assets/LeapMotion/Core/Scripts/LeapDeviceInfo.cs

```diff
@@ -57,7 +57,7 @@ public struct LeapDeviceInfo {
 
       LeapDeviceInfo deviceInfo;
       deviceInfo.isEmbedded = false;
-      deviceInfo.baseline = 0.04f;
+      deviceInfo.baseline = 0.064f;
```

Comment:
If we are still hardcoding this, we should at least hardcode it to the value used by peripheral (0.04) since that is what developers are going to experience.

Reply:
I definitely don't want to hard-code this anymore, but let's address that as part of the Image API PR, yeah?

Changing it back to 0.04f in the meantime